x/stakeibc/keeper/msg_server.go

```diff
@@ -1001,6 +1001,21 @@ func (k msgServer) RestoreInterchainAccount(goCtx context.Context, msg *types.Ms
 	return &types.MsgRestoreInterchainAccountResponse{}, nil
 }
 
+// Admin transaction to close an ICA channel
+// This can be used if there are records stuck in state IN_PROGRESS after a channel has been re-opened after a timeout
+// After the closure, the a new channel can be permissionlessly re-opened with RestoreInterchainAccount
+func (k msgServer) CloseDelegationChannel(goCtx context.Context, msg *types.MsgCloseDelegationChannel) (*types.MsgCloseDelegationChannelResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	_, capability, err := k.IBCKeeper.ChannelKeeper.LookupModuleByChannel(ctx, msg.PortId, msg.ChannelId)
+	if err != nil {
+		return nil, errorsmod.Wrap(err, "could not retrieve capability from port ID and channel ID")
+	}
+	if err := k.IBCKeeper.ChannelKeeper.ChanCloseInit(ctx, msg.PortId, msg.ChannelId, capability); err != nil {
```

Comment:
is there a case where chancloseinit fails or gets stuck in a pending state? if so would we just re-send this `CloseDelegationChannel` tx?

Reply:
i don't think so cause it goes straight to state CLOSED on stride (there's not handshake process like with opening)

Either way, not relevant anymore now that we're doing the send packet instead